source/conf.py

```diff
@@ -135,7 +135,7 @@
     "use_repository_button": True,
     "use_issues_button": True,
     "use_edit_page_button": True,
-    "announcement": "This is the new Open edX documentation root site and it is currently under development.",
+    "announcement": "This is the new documentation homepage for the Open edX project. It's currently under development, but feel free to take a look around.",
```

Comment:
Is "homepage" used anywhere else in the site? I know we used index for the names of the root page, but for consistency are we calling this the documentation homepage everywhere?